diff --git a/devices/planet-geminipda/kernel/config b/devices/planet-geminipda/kernel/config
index 36e33c0..92b30f1 100644
--- a/devices/planet-geminipda/kernel/config
+++ b/devices/planet-geminipda/kernel/config
@@ -9669,7 +9669,7 @@ CONFIG_QUOTA=y
 # CONFIG_QFMT_V2 is not set
 CONFIG_QUOTACTL=y
 CONFIG_AUTOFS_FS=y
-# CONFIG_FUSE_FS is not set
+CONFIG_FUSE_FS=y
 CONFIG_CUSE=m
 # CONFIG_VIRTIO_FS is not set
 # CONFIG_OVERLAY_FS is not set
